bokehjs/src/lib/core/util/svg.ts

```diff
@@ -622,7 +622,11 @@ export class SVGRenderingContext2D /*implements CanvasRenderingContext2D*/ {
     if (!isFinite(x + y))
       return
 
-    if (this.__currentElement.nodeName !== "path") {
+    /*
+     * If there is no current element or this is the first element or
+     * current element is no a path
+    */
```

Comment:
This comment is superfluous, it just restates exactly what the line below does, verbatim. If there is a need to explain *why* something happens, then a useful comment will explain the "why". But "what" and "how" comments are only needed if some following code is irreducibly complicated in some way.

Reply:
it makes sense, thanks bryan!